src/views/ajax.js

```diff
@@ -0,0 +1,112 @@
+const util = require('../lib/util');
+const dom = require('../lib/dom');
+
+let count = 0;
+let ready = false;
+let preRenderCache = [];
+let currentTimout = undefined;
+const summaryStack = [];
+
+function rowTemplate(method, uri, duration) {
```

Comment:
Suggestion: have all these render functions refactored to be like:

```js
function rowTemplate({ method, uri, duration }) {
  // ...
}
```

which will make refactoring (and type checking) much easier in the future 😄